### PR TITLE
Spell checking: fixed typos

### PR DESCRIPTION
**Pull request by neteler**
Fixed typos in entire source code tree using `utils/fix_typos.sh`:

- typos fixed in C/CPP, Python, HTML, Markdown, other text files
- gettext messages updated accordingly to not break translations
- gettext `.pot` files regenerated:
    -  `cd locale/po/`
    - `make pot`
- `.po` files trimmed for (now) duplicate messages and unused fuzzy messages:
    -  `cd locale/po/`
    - `for i in `ls gr*.po` ; do POFILE=$i ; msgattrib --output-file=$i --no-obsolete $POFILE ; done`
- updated `utils/fix_typos.sh`

### CODE REVIEW

File: gui/wxpython/core/render.py [hunk 1]
```diff
@@ -542,7 +542,7 @@ def __init__(self, Map):
     def _init(self):
         """Init render manager"""
         self._startTime = time.time()
-        self.progressInfo = None
+        self.progresInfo = None
```
Comment (neteler) [inline gui/wxpython/core/render.py:545]:
```suggestion
        self.progressInfo = None
```

File: gui/wxpython/core/render.py [hunk 2]
```diff
@@ -763,7 +763,7 @@ def ReportProgress(self, env, layer=None):
         """Calculates progress in rendering/downloading
         and emits signal to inform progress bar about progress.
 
-        Emits renderDone event when progressVal is equal to range.
+        Emits renderDone event when progresVal is equal to range.
```
Comment (neteler) [inline gui/wxpython/core/render.py:766]:
```suggestion
        Emits renderDone event when progressVal is equal to range.
```

File: lib/vector/dglib/README [hunk 3]
```diff
@@ -60,7 +60,7 @@
  *
  * Field                          Size/Type                     Value range          Description
  * [VERSION...................]   1 byte                        1                    Version 1 graphs always keep value 1
- * [ENDIANESS.................]   1 byte                        1 | 2                Integer words byte order 1=big 2=little
+ * [ENDIANNESS.................]   1 byte                        1 | 2                Integer words byte order 1=big 2=little
```
Comment (neteler) [inline lib/vector/dglib/README:63]:
```suggestion
 * [ENDIANNESS.................]   1 byte                       1 | 2                Integer words byte order 1=big 2=little
```

**Review by nilason — APPROVED** [review]
Great work! Much needed.

Note: I will address the corrections in ctypesgen upstreams too.

**Pull request merged**